app/controllers/account_requests_controller.rb

```diff
@@ -24,7 +24,7 @@ def new
   def create
     @account_request = AccountRequest.new(account_request_params)
 
-    if @account_request.save
+    if verify_recaptcha(model: @account_request) && @account_request.save
```

Comment:
@esquith would this cause registration to not function locally? If so, maybe we can conditionally require this using `Rails.env.production?` and/or `Rails.env.staging?` so that it only shows up in deployed environments. It isn't something we need when testing locally.

Reply:
Thanks, will change this

Reply:
Note: Google [provides test API keys](https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do) for local verification and automated testing. When using Recaptcha with these, the button click captcha solve _always_ succeeds. If it doesn't add additional setup effort for a contributor, it might be worth having so that we see the same user flow in dev/test as in prod.

I'll experiment a little with using that on this branch.